File: py/day-2/first_solution.py
def solution(inputArr):
  h=0
  v=0
  for input in inputArr:
    c = input.split(' ')
    d = c[0]
    val = int(c[1])
    if (d == 'forward'):
      h += val
    elif(d == 'up'):
       v -= val
    elif(d == 'down'):
      v += val
  return h*v

File: js/day-2/first-solution.js
module.exports = (arr) => {
  let h = 0; let v = 0
  arr.map((i) => {
    const c = i.split(' ')
    const d = c[0]
    const val = +c[1]
    if (d == 'forward') {
      h += val
    } else if (d == 'up') {
      v -= val
    } else if (d == 'down') {
      v += val
    }
  })

  return h * v
}
